src/index.html

```diff
@@ -1,13 +1,64 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no,   initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+
+  <body>
+    <div
+      class="container"
+      data-qa="card"
+    >
+
+      <div class="product-card">
+        <img
+          class="main-image"
+          src="images/imac.jpeg"
+          alt="product-image-imac"
+        >
+
+        <h1 class="product-name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h1>
+
+        <p class="product-code">Код товара: 195434</p>
+
+        <div class="feedback">
+
+          <div class="ratings">
+            <div class="stars stars--4">
```

Comment:
```suggestion
            <div class="stars stars-4">
```
через "--" обозначается модификатор элемента

Reply:
Yep